content/docs/integration/integrate-with-apps@id.md

```diff
@@ -0,0 +1,199 @@
+---
+$title: Mengintegrasikan AMP dengan aplikasi Anda 
+$order: 6
+$category: Develop
+toc: true
+
```

Comment:
Please remove the added frontmatter from all the files. Only the translated $title should be present on translated files. Thank you.

Reply:
Updated all the translation files with the front matter changes.